apps/nightscout/nightscout.star

```diff
@@ -205,7 +433,11 @@ def get_nightscout_data(nightscout_id):
 
     # If it's not in the cache, construct it from a response.
     print("Miss - calling Nightscout API")
-    nightscout_url = "https://" + nightscout_id + ".herokuapp.com/api/v1/entries.json"
+
+    #nightscout_url = "https://" + nightscout_id + "/api/v1/entries.json?count="+str(GRAPH_WIDTH)
+    nightscout_url = "https://" + nightscout_id + "/api/v1/entries/sgv.json?find[date][$gte]=" + str((OLDEST_READING_TARGET.unix) * 1000) + "&count=" + str(GRAPH_WIDTH)
```

Comment:
I’d argue that you couldn’t point this to any json feed because you’d need
fields named svg and date. The reason the current implementation doesn't work is because of the reasons I enumerated in the comment above, namely that not all Nightscout instances are at Heroku.